extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -384,6 +384,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       'rc': this.fromResumeCallback ? 1 : null,
       'rafmt': this.getRafmtParam_(),
       'pfx': pfx ? '1' : '0',
+      'aanf': this.element.getAttribute('data-no-fill'),
```

Comment:
Given a pub could theoretically set this value to anything and pass validation, let's make sure it's a boolean.  Can you change this to:

'aanf': /^(true|false)$/i.test(this.element.getAttribute('data-no-fill')) ? this.element.getAttribute('data-no-fill')  : null,

Please add test that ensures only those values are allowed (not case insensitivity).

Reply:
Done. Thanks!!